.github/workflows/commit.yml

```diff
@@ -0,0 +1,79 @@
+---
+name: commit
+
+on:
+  push:
+
+
+jobs:
+  test:
+    name: test
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: run-tests
+        run: make cloud-test
```

Comment:
The CircleCI configuration allowed `make integration-test` and `make unit-test` to run in parallel. Using one target in one step here means they have to run serially. Not a big deal since they're relatively quick tests, but this is something that could be improved in future.